src/views/v2/components/events-bar/filters.php

```diff
@@ -1,15 +0,0 @@
-<?php
```

Comment:
Same as above, removing this file will create issues with back-compatibility, intentional?